presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraTokenSplitManager.java

```diff
@@ -27,41 +27,44 @@
 {
     private static final int SPLIT_SIZE = 100;
     private static final String KEYSPACE = "test_cassandra_token_split_manager_keyspace";
-    private static final String TABLE = "test_cassandra_token_split_manager_table";
     private static final int PARTITION_COUNT = 1000;
 
     private CassandraSession session;
     private CassandraTokenSplitManager splitManager;
 
-    @BeforeMethod
+    @BeforeClass
     public void setUp()
             throws Exception
     {
         EmbeddedCassandra.start();
         session = EmbeddedCassandra.getSession();
+        createKeyspace(session, KEYSPACE);
         splitManager = new CassandraTokenSplitManager(session, SPLIT_SIZE);
     }
 
     @Test
-    public void testCassandraTokenSplitManager()
+    public void testEmptyTable()
             throws Exception
     {
-        createKeyspace(session, KEYSPACE);
-        session.execute(format("CREATE TABLE %s.%s (key text PRIMARY KEY)", KEYSPACE, TABLE));
-
-        EmbeddedCassandra.refreshSizeEstimates(KEYSPACE, TABLE);
-
-        List<TokenSplit> splits = splitManager.getSplits(KEYSPACE, TABLE);
+        String tableName = "empty_table";
+        session.execute(format("CREATE TABLE %s.%s (key text PRIMARY KEY)", KEYSPACE, tableName));
```

Comment:
Due to the "eventual" consistency of Cassandra i don't see a point why do i need to drop it. Anyway it coudn't be possible to run this test in a loop. And after this test has finished - the entire in-memory Casandra cluster will be wiped out.